uber/site_sections/accounts.py

```diff
@@ -19,7 +19,7 @@ def index(self, session, message=''):
             'all_attendees': sorted([
                 (id, '{} - {}{}'.format(name.title(), BADGES[badge_type], ' #{}'.format(badge_num) if badge_num else ''))
                 for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num)
-                                                              .filter(Attendee.email != '').all()
+                                                              .filter(Attendee.email != '').filter(Attendee.status != INVALID_STATUS).all()
```

Comment:
I have no working AC dev setup. How fast do you need it done?

--
Robert Earl <r.a.earl@gmail.com>
FAX: 1-440-919-5086

On Thu, Oct 8, 2015 at 1:34 PM, Victoria Earl <notifications@github.com>
wrote:

> In uber/site_sections/accounts.py
> <https://github.com/RAMSProject/rams/pull/3#discussion_r41543216>:
>
> > @@ -19,7 +19,7 @@ def index(self, session, message=''):
> >              'all_attendees': sorted([
> >                  (id, '{} - {}{}'.format(name.title(), BADGES[badge_type], ' #{}'.format(badge_num) if badge_num else ''))
> >                  for id, name, badge_type, badge_num in session.query(Attendee.id, Attendee.full_name, Attendee.badge_type, Attendee.badge_num)
> > -                                                              .filter(Attendee.email != '').all()
> > +                                                              .filter(Attendee.email != '').filter(Attendee.status != INVALID_STATUS).all()
>
> @earl7399 <https://github.com/earl7399> Can you add this and submit a PR
> for it? We only want NEW and COMPLETED attendees in the accounts list,
> actually. Don't forget that the column name changed to badge_status.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RAMSProject/rams/pull/3/files#r41543216>.
>